main/guides/getting-started/README.md

```diff
@@ -43,14 +47,24 @@ The Agoric SDK is supported on <a href="https://en.wikipedia.org/wiki/Linux">Lin
 - To launch a terminal on Linux, use the **Terminal** application.
 - To access WSL from Windows, visit the [WSL documentation](https://docs.microsoft.com/en-us/windows/wsl/).
 
-## Install Node.js 16.19.1 or Higher
+## Install Go
 
-Download Node.js from [Node.js](https://nodejs.org/) and follow the instructions for your platform. We recommend installing the LTS version of node 16.
+Download Go from [go.dev/doc/install](https://go.dev/doc/install) and follow the instructions for your platform.
 
 ```shell
-node --version # 16.19.1 or higher
+go version # Version 1.20.3 or higher
 ```
 
+## Install Node.js
+
+Download Node.js from [Node.js](https://nodejs.org/) and follow the instructions for your platform. We recommend installing the LTS version of node 18.
+
+```shell
+node --version # LTS version such as 18.16.0
+```
+
+**Note:** Agoric will support all even numbered versions of Node.js. 
```

Comment:
**important**:

```suggestion
**Note:** Agoric will support all long-term support (LTS) versions of Node.js. 
```

not all even numbered versions are always LTS versions. first node 18.1.0 comes out, then folks notice problems, then 18.2.0... by around 18.5.0, problem reports slow down and they put the LTS stamp on it.

Reply:
Ah. So odd numbered versions don't get the LTS stamp and only certain iterations of even numbered versions get the LTS stamp? Gotcha.

Reply:
Oh... and the text was changed to match your suggestion.